zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/AbstractDependencyResolver.java

```diff
@@ -64,8 +64,8 @@ public void addRepo(String id, String url, boolean snapshot) {
     synchronized (repos) {
       delRepo(id);
       RemoteRepository rr = new RemoteRepository(id, "default", url);
-      rr.setPolicy(snapshot, new RepositoryPolicy(
-          true,
+      rr.setPolicy(true, new RepositoryPolicy(
```

Comment:
According to [sonatype aether api doc](http://sonatype.github.io/sonatype-aether/apidocs/org/sonatype/aether/repository/RemoteRepository.html#setPolicy(boolean, org.sonatype.aether.repository.RepositoryPolicy)), original code seems to be the correct. Did I miss something?